jenkins/bootstrap.py

```diff
@@ -365,7 +365,7 @@ def upload_json(self, path, jdict, generation=None):
         else:
             gen = []
         with tempfile.NamedTemporaryFile(mode='wt', encoding='utf-8', prefix='gsutil_') as fp:
-            json.dumps(jdict, fp, indent=2)
```

Comment:
https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/115322/pull-kubernetes-node-e2e-containerd/1618413835490693120/

the `s` here is for `str` ...